control-plane-operator/controllers/hostedcontrolplane/scheduler/deployment.go

```diff
@@ -166,6 +166,7 @@ func schedulerVolumeKubeconfig() *corev1.Volume {
 
 func buildSchedulerVolumeKubeconfig(v *corev1.Volume) {
 	v.Secret = &corev1.SecretVolumeSource{
-		SecretName: manifests.SchedulerKubeconfigSecret("").Name,
+		SecretName:  manifests.SchedulerKubeconfigSecret("").Name,
+		DefaultMode: pointer.Int32Ptr(0640),
```

Comment:
nit: it'd be good to have this in a constant so if we ever change it, don't need to through all places.

Reply:
@enxebre thank you, i will look in to that and also can you apply the lgmt label